tgui/packages/tgui/interfaces/APC.js

```diff
@@ -0,0 +1,195 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Box, Button, LabeledList, NoticeBox, ProgressBar, Section } from '../components';
+import { Window } from '../layouts';
+import { InterfaceLockNoticeBox } from './common/InterfaceLockNoticeBox';
+
+export const APC = (props, context) => {
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
```

Comment:
I feel like this UI shouldnt need scrolling on it or the ability to resize, it doesnt have any content which will require expansion via scrolling, and the height can just be a preset height thats available to include everything.

TLDR: Scrollbars when scrolling isnt needed looks bad and wastes a bit of horizontal space.